base/ca/src/main/java/com/netscape/ca/CAService.java

```diff
@@ -1716,7 +1719,13 @@ private boolean compareChallengePassword(CertRecord record, String pwd)
             throw new EBaseException(CMS.getUserMessage("CMS_BASE_INVALID_ATTRIBUTE", "metaInfo"));
         }
 
-        String hashpwd = hashPassword(pwd);
+        String hashpwd = null;
+        try {
+            hashpwd = hashPassword(pwd);
+        } catch (NotInitializedException e) {
+            logger.error("CryptoManager is not initialized, can't compare password so failing challenge");
+            return false;
```

Comment:
JSS should be initialized at startup, so if we get this exception that means something went wrong. I think it would be better to wrap and rethrow the exception so that the caller knows there's something wrong instead of getting a negative password validation:
```
throw new EBaseException(e);
```

Reply:
Done.